src/modules/PackageSlideShow.rb

```diff
@@ -8,17 +8,18 @@ class PackageSlideShowClass < Module
     include Yast::Logger
 
     # seconds to cut off predicted time
-    MAX_TIME_PER_CD = 7200
-
-    # minimum time displayed per CD if there is something to install
-    MIN_TIME_PER_CD = 5
+    MAX_TIME = 2 * 60 * 60 # 2 hours
 
     # Column index for refreshing statistics: remaining size
     SIZE_COLUMN_POSITION = 1
     # Column index for refreshing statistics: remaining number of packages
     PKG_COUNT_COLUMN_POSITION = 2
     # Column index for refreshing statistics: remaining time
     TIME_COLUMN_POSITION = 3
+    # Table padding
+    ITEM_PREFIX = " " * 4
+    # hourglass unicode char
+    HOURGLASS = "\u231B".freeze
```

Comment:
Ruby uses by default UTF-8 for the sources, so we could use the literal character here. The advantage is that you can directly see how that looks like.

```suggestion
    HOURGLASS = "⌛".freeze
```

Or is there any reason for using the escape sequence?

Reply:
Well, reason for me is that it depends on used font how it will in the end look like. So I prefer have something that can be googled ( I know that even unicode char can be goodled ) and also something that is obvious that it is utf-8 char ( easy for this, but not so for e.g. … )